code/gam-model.R

```diff
@@ -148,8 +148,6 @@ plot(pedal_cycles ~ year, data = forplot)
 
 # Fit GAM model using national grid locations--------------------------------------------
 
-## should I use peak hour counts only for these models, to match with census commute data??
```

Comment:
I think using all hours makes sense, we can ask for model results for those days we have readings though...